app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -69,7 +69,9 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
       if ((lastUpdate.status == GithubGoldStatusUpdate.statusCompleted &&
               lastUpdate.head == pr.head.sha) ||
           (pr.base.ref != 'master')) {
-        log.debug('Completed status already reported for this commit.');
+        log.debug(pr.base.ref == 'master'
```

Comment:
drive by nit: this whole code would be more readable by changing line 69 to an `if`...`else if`, with each having an explicit log message.